### DISCUSSION TITLE
I am a beginner and I want to start contributing. Please help me out.

### DISCUSSION THREAD

**Discussion by shlokjjw** [cncf/mentoring#620] [Q&A] [answered]
Hi, I am Shlok Jhunjhunwala and I want to contribute in CNCF projects.
Can someone guide me from where I can get started. 

I have knowledge in MERN stack and have beginner level knowledge in k8s, docker, maven, jenkins, ansible and nagios.
I want to implement my knowledge and contribute. 

Your guidance will be of great support to me.

**Comment by nate-double-u** [comment, marked as answer]
Hi @shlokjjw!

If you want to be a Cloud Native contributor, I suggest starting by checking out the [CNCF Contributors](https://contribute.cncf.io/) page.

If you're interested in mentorship, the CNCF is participating in GSoC, and you can find info in the [Google Summer of Code 2022 & CNCF](https://github.com/cncf/mentoring/discussions/565) discussion thread.
